modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_docker_container_upload.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'pdf_generator_service/pdf_client'
+require 'claims_api/v2/disability_compensation_evss_mapper'
+require 'evss_service/base'
+
+module ClaimsApi
+  module V2
+    class DisabilityCompensationDockerContainerUpload < DisabilityCompensationClaimServiceBase
+      def perform(claim_id) # rubocop:disable Metrics/MethodLength
+        log_job_progress('526 v2 Docker Container job',
+                         claim_id,
+                         'Docker container job started')
+
+        auto_claim = get_claim(claim_id)
+        # Reset for a rerun on this
+        set_pending_state_on_claim(auto_claim) unless auto_claim.status == pending_state_value
+
+        if auto_claim.evss_id.nil?
+          evss_data = evss_mapper_service(auto_claim, veteran_file_number(auto_claim)).map_claim
+
+          log_job_progress('526 v2 Docker Container job',
+                           claim_id,
+                           'Submitting to Docker container')
+
+          evss_res = evss_service.submit(auto_claim, evss_data)
+
+          log_job_progress('526 v2 Docker container job',
+                           claim_id,
+                           "Successfully submitted to Docker container with response: #{evss_res}")
```

Comment:
just a question for clarity: any concerning data in `evss_res` here? we're not going to have PII here, right?

Reply:
@FonzMP  The evss_res looks something like this `:claimId=>600150327, :endProductClaimCode=>\"400SUPP\", :endProductClaimName=>\"eBenefits 526EZ-Supplemental (400)\", :inflightDocumentId=>394121}`